source/parser.hera

```diff
@@ -6524,8 +6524,10 @@ Shebang
   /#![^\r\n]*/ EOL
 
 CivetPrologue
-  [\t ]* DoubleQuote CivetPrologueContent:content DoubleQuote SimpleStatementDelimiter EOS? -> content
-  [\t ]* SingleQuote CivetPrologueContent:content SingleQuote SimpleStatementDelimiter EOS? -> content
+  [\t ]* DoubleQuote CivetPrologueContent:content DoubleQuote SimpleStatementDelimiter _? EOL? EOS?:eos ->
```

Comment:
Why did you add `_? EOL?`? In particular, this will discard inline comments (in `_`), and seems like it can consume two newlines. But maybe I'm missing something?

Reply:
Oh interesting! I overlooked that `_` included InlineComment.

With `_? EOL?` I was looking for a simple way to do these 2 things:

1. Trim initial whitespace from an end-of-line comment (or the start of a multiline one)
2. Omit the blank line if there's no comment after a directive

So that

```coffee
"civet -implicitReturns" // hello!
"civet autoLet"
// hi!
```

becomes

```ts
// hello!
// hi!
```

rather than 
```ts
 // hello!

// hi!
```

Reply:
Ah, I see. Thanks for the explanation! That makes sense now.

The `Init` rule where this gets used has (recently added) support for comments, so I wonder if the following would actually be a simpler solution to this problem:

```
... PrologueEOL [ \t]* ( EOL / &RestOfLine ) -> content
```

This should leave the comments on the remainder of the line for further processing, but only if there's something other than a newline, after trimming spaces/tabs.